test/agents/test_tools_manager.py

```diff
@@ -16,6 +16,14 @@ def tools_manager():
     return ToolsManager(tools=tools)
 
 
+@pytest.mark.unit
+def test_callable_tool():
+    # test that we can also pass a callable as a tool
```

Comment:
```suggestion
def test_callable_pipeline_tool():
    # test that we can also pass a callable as the pipline of a tool
```

Reply:
Hi @omerfarukabaci, thank you for your comments! I resolved your other comment, and for this one, I leave it to @bogdankostic to decide :)

Reply:
Hi @bilgeyucel, thank you for resolving the previous issue. I'm looking forward to @bogdankostic's decision on this one.

Reply:
The most explicit name I can come up with would be `test_using_callable_as_tool`.